lib/rules/function-calc-no-unspaced-operator/index.cjs

```diff
@@ -358,15 +252,16 @@ const rule = (primary, _secondaryOptions, context) => {
 
 						// Step 4
 						// If there is no operator, try to find one
-						if (isOperationWithoutOperator(operation)) {
+						if (!operation.operator) {
 							checkOperationWithoutOperator(node, operation);
 						}
 
 						// Step 5
 						// If the operation is complete, ensure there is whitespace around the operator
 						// The operation might have started without an operator and might have been repaired by Step 4
```

Comment:
minor
question

> and might have been repaired by Step 4

Is this still true?

Reply:
yes :)